### ISSUE TITLE
High vulnerability in dependency (lodash.merge)

### ISSUE THREAD

**Issue by gorner** [gabrielgrant/broccoli-less-single#47]
`yarn audit` output includes the following:

```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ high          │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash.merge                                                 │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.6.2                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ [other package]                                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ [other packages] > broccoli-less-single > lodash.merge       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/1066                        │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

The package.json specifies `"lodash.merge": "^3.3.2"` so (assuming the updated version is compatible) a major version bump is required to overcome.

**Comment by txm** [comment]
https://github.com/gabrielgrant/broccoli-less-single/pull/49

**Comment by jasonmit** [comment]
Resolved in 1.1.0 - thanks!
